examples/complete/main.tf

```diff
@@ -94,6 +94,18 @@ module "full_example" {
             "index.html" = base64encode("<h1>Hello World</h1>")
           }
         }
+        secrets = {
+          mount_path = "/etc/secrets"
+          secret = {
+            "username" = base64encode("foobar")
+          }
+          secret_from_key_vault = {
+            credentials = {
+              key_vault_id = "/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.KeyVault/vaults/<KEY_VAULT>"
```

Comment:
can you actually add a KeyVault (there's a GiD module for that) in this example, add a secret and use that in the `secret_from_key_vault`? I really want `complete` examples to be runnable for instance for testing purposes. Having hardcoded fake `key_vault_id` makes this example unrunnable.